pkg/cmd/context/context.go

```diff
@@ -1,6 +1,7 @@
 package context
 
 import (
+	namespaceUse "github.com/redhat-developer/app-services-cli/pkg/cmd/connector/namespace/use"
```

Comment:
There's a `context unset` command, would we like to have namespace and connector in unset?

Reply:
+1